File: Source/Rewards/Rewards/ChampionAwards.cs
using System.Linq;
using WCSharp.Api;

public static class ChampionAwards
{
    public static void AwardAllChampions()
    {
        foreach (var player in Globals.ALL_PLAYERS)
        {
            if (!Globals.CHAMPIONS.Contains(player.Name)) continue;
            GiveAllChampionAwards(player);
        }
    }

    private static void GiveAllChampionAwards(player player)
    {
        Tournament awards;
        AwardManager.GiveReward(player, nameof(awards.TurquoiseNitro), false);
        AwardManager.GiveReward(player, nameof(awards.TurquoiseWings), false);
        AwardManager.GiveReward(player, nameof(awards.VioletAura), false);
        AwardManager.GiveReward(player, nameof(awards.VioletWings), false);
        AwardManager.GiveReward(player, nameof(awards.PenguinSkin), false);
    }

}

File: src/Rewards/Rewards/ChampionAwards.ts
import { Globals } from "src/Global/Globals"
import { Tournament } from "src/SaveSystem2.0/MAKE REWARDS HERE/SaveObjects/RewardObjects/Tournament"
import { MapPlayer } from "w3ts"
import { AwardManager } from "./AwardManager"

export class ChampionAwards {
    public static AwardAllChampions() {
        for (let player of Globals.ALL_PLAYERS) {
            if (!Globals.CHAMPIONS.includes(player.name)) continue
            this.GiveAllChampionAwards(player)
        }
    }

    private static GiveAllChampionAwards(player: MapPlayer) {
        let awards: Tournament
        AwardManager.GiveReward(player, 'TurquoiseNitro', false)
        AwardManager.GiveReward(player, 'TurquoiseWings', false)
        AwardManager.GiveReward(player, 'VioletAura', false)
        AwardManager.GiveReward(player, 'VioletWings', false)
        AwardManager.GiveReward(player, 'PenguinSkin', false)
    }
}
